### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by MichelleSwafford**
Updating CODEOWNER with Staged Rollout contacts

## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
* [ ] *if updating templates, it is helpful to post a screenshot of what the changes look like.*
* [ ] *if updating templates, ensure that your parameters and steps have meaningful names.*
* [ ] *if adding new templates, ensure that your parameters and columns have display names.*
* [ ] *if adding new templates, or changing items in a gallery, it's helpful to post a screenshot of what the gallery looks like now*

### CODE REVIEW

File: CODEOWNERS
Comment (gardnerjr) [inline CODEOWNERS:29]: this would make you the *only* owner.  since you can't approve your own changes, and at least one owner would be required, you'd create a case where everyone *except* you can request a review but you're the only approver, so you yourself could never make changes to these files since an owner is required to approve.  if this is AAD related, that microsoft/azure-ad-workbooks "team" should be the owner for AAD related workbooks, so that your whole team can approve changes here

**Review by gardnerjr — CHANGES_REQUESTED** [review]
you didn't need to make a new PR for this, you could have added this to your existing PR.  as is right now, this would block your other PR since you'd be the only person allowed to approve but you can't approve your own changes

**Comment by MichelleSwafford** [comment]
Are there any additional changes required before this can be approved?

**Comment by MichelleSwafford** [comment]
Can this be checked-in?

**Comment by danielwood95** [comment]
Michelle is a member of @azure-ad-worbooks so she doesn't need to be individually added to CODEOWNERS.

**Comment by gardnerjr** [comment]
you closed it, but the PR here didn't add JUST michelle, it added codeowners _entirely_.  so like i said in the other PR, unless THAT PR updates codeowners (it doesn't!), then this PR needs to be completed.  One of the 2 PR's has to update CODEOWNERS so the new template has code owners set up right from the start.  if she doesn't explicitly need to be on the list, then the PR should be edited to remove her but leave the team ownership.

**Pull request merged**
